Src/WitsmlExplorer.Api/Workers/OffsetLogCurveWorker.cs

```diff
@@ -176,7 +177,11 @@ private async Task DeleteLogData(WitsmlLog log, WitsmlLogCurveInfo logCurveInfo,
 
         private async Task UpdateLogData(WitsmlLog log, WitsmlLogCurveInfo logCurveinfo, WitsmlLogData offsetLogData)
         {
-            var queries = GetUpdateLogDataQueries(log, offsetLogData);
+            var mnemonics = log.LogCurveInfo.Select(lci => lci.Mnemonic).ToList();
```

Comment:
Can you use the mnemonicList below instead of using lci? The log can have more lci elements than what we actually send in a query, so using the list from the data should be more accurate.